cmds/addTimeEntry.mjs

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-unused-expressions */
+import Client from '../client.js'
+import { defaultWorkspaceId, getProjectByName, getProjectById, appName, displayTimeEntry, parseTime } from '../utils.js'
+import dayjs from 'dayjs'
+import debugClient from 'debug'
+import utc from 'dayjs/plugin/utc.js'
+import timezone from 'dayjs/plugin/timezone.js'
+import yargs from 'yargs'
+dayjs.extend(utc)
+dayjs.extend(timezone)
+
+const debug = debugClient('toggl-cli-add');
+
+export const command = 'add <startTime> <endTime> <description>'
```

Comment:
I think these should be optional using the `[]` syntax, so that the `start`, `end` and `description` arguments can be used, too.

This will give us more flexibility - I think about adding new entries like "I did xxx from start until end", but it wil work in the "From start to end, I did xxx"

I'll make that change and test - but I'm in the middle of making brunch and I discovered that I can push changes!

Reply:
With the change in 

this works

`DEBUG=toggl-cli-add ./cli.js add -d foo -s 8:30 -e '9:02 AM'`

as well as what you had originally 

`DEBUG=toggl-cli-add ./cli.js add 8:00 '08:33 AM' foo`

Reply:
Great, glad we have functionality we both like!